### DISCUSSION TITLE
What's the syntax highlighter github using?

### DISCUSSION THREAD

**Discussion by Thecode764** [community/community#132476] [Other Feature Feedback, Questions, & Ideas] [answered]
Please read title

for example

```python
print("Hello world")
```

**Comment by NirukshaSandeepa** [comment, marked as answer]
GitHub uses [linguist](https://github.com/github-linguist/linguist) to perform language detection and to select [third party grammars](https://github.com/github-linguist/linguist/blob/master/vendor/README.md) for syntax highlighting.

**Reply by Thecode764** [reply]
thank you

**Reply by Thecode764** [reply]
but i using react :((
